requirements/base.txt

```diff
@@ -36,7 +36,7 @@ django-autoslug==1.8.0
 pytz==2012h
 
 # background process management
-# django-celery==3.1.16
+django-celery==3.1.16
```

Comment:
I followed docs/source/LINUX.setup.rst to deploy SEED, if django-celery is
missing, command 'python manage.py celery worker -B -c 2 --loglevel=INFO -E
--maxtasksperchild=1000' will return 'command celery is not found' error.
Do we have a new command to run Celery?

On Thu, Mar 3, 2016 at 4:44 PM, Nicholas Serra <notifications@github.com>
wrote:

> In requirements/base.txt
> <https://github.com/SEED-platform/seed/pull/817#discussion_r54953401>:
>
> > @@ -36,7 +36,7 @@ django-autoslug==1.8.0
> >  pytz==2012h
> >
> >  # background process management
> > -# django-celery==3.1.16
> > +django-celery==3.1.16
>
> We're running the celery package straight now, no need for django-celery,
> this shouldn't be uncommented.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SEED-platform/seed/pull/817/files#r54953401>.
>